README.md

```diff
@@ -118,6 +118,49 @@ example :
   - your-page-doc-name
 ```
 
+#### Create a release
```

Comment:
Level 4 means it would be a subtitle of `Project Set up > Build and run the documentation website`
I think it is more linked to the `Publication` title, so I would put it under it.